components/Carousel/Carousel.js

```diff
@@ -1,14 +1,75 @@
 class Carousel {
```

Comment:
great job on carousel and keeping the logic clean for keeping a proper rotation of images!